code-blast.js

```diff
@@ -225,33 +225,36 @@ https://twitter.com/JoelBesada/status/670343885655293952
 			canvas.style.position = 'fixed';
 			canvas.style.top = 0;
 			canvas.style.left = 0;
-			canvas.style.zIndex = 1;
+			canvas.style.zIndex = 2147483647;
```

Comment:
if ace is in a popup, which has higher z-index than canvas, canvas will be behind the editor, and will not be visible.